libs/ui-lib/lib/common/config/docs_links.ts

```diff
@@ -18,48 +18,47 @@ export const CLUSTER_MANAGER_SITE_LINK = 'https://console.redhat.com/openshift/i
 export const PULL_SECRET_INFO_LINK = CLUSTER_MANAGER_SITE_LINK;
 
 export const getEncryptingDiskDuringInstallationDocsLink = (ocpVersion?: string) =>
-  `https://docs.openshift.com/container-platform/${
-    getShortOpenshiftVersion(ocpVersion) || '4.12'
-  }/installing/install_config/installing-customizing.html#installation-special-config-encrypt-disk_installing-customizing`;
+  `https://docs.redhat.com/en/documentation/openshift_container_platform/${
+    getShortOpenshiftVersion(ocpVersion) || '4.13'
```

Comment:
In the subtasks, they generally refer to version 4.15, so I think this should be the default one.
But ideally, it should also be a constant rather than being hardcoded in several of the links
```suggestion
    getShortOpenshiftVersion(ocpVersion) || DEFAULT_OPENSHIFT_DOCS_VERSION
```